frappe/www/message.html

```diff
@@ -24,7 +24,7 @@ <h5 class='page-card-head'>
 		<span class='indicator {{ indicator_color or "blue" }}'>
 			{{ title or _("Message") }}</span>
 	</h5>
-	<div class="page-card-body ellipsis">
+	<div class="page-card-body">
 	{% block message_body %}
 		<p>{{ message or "" }}</p>
 		{% if primary_action %}
```

Comment:
`ellipsis` might be required in some cases to avoid text overflow. How about adding it to `<p>`
```suggestion
	<div class="page-card-body">
	{% block message_body %}
		<p class="ellipsis">{{ message or "" }}</p>
		{% if primary_action %}
```

Reply:
So, I've checked this, the ellipsis is not really required. With our without it there the content does not overflow the container.